<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登陆界面</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QLabel" name="titleLb">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>45</y>
      <width>300</width>
      <height>10</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="accoutEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="accoutLb">
       <property name="text">
        <string>账号：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="passwordLb">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nameLb">
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>159</y>
      <width>166</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="signupBtn">
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="loginBtn">
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
